src/plots/linePlot.ts

```diff
@@ -106,6 +128,7 @@ export module Plots {
         return d3.svg.line()
                      .x((innerDatum, innerIndex) => xProjector(innerDatum, innerIndex, dataset))
                      .y((innerDatum, innerIndex) => yProjector(innerDatum, innerIndex, dataset))
+                     .interpolate(this.interpolate())
```

Comment:
Just double checking, the linear interpolation is our current non-settable default? (prior to this patch)